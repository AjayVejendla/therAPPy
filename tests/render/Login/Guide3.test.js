import React from "react";
import Guide3 from "../../../screens/Login/Guide3";
import renderer from "react-test-renderer";

test("Guide3 render test", () => {
  const tree = renderer.create(<Guide3 />).toJSON();
  expect(tree).toMatchSnapshot();
});
