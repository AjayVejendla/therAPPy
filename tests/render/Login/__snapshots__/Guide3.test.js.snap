// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Guide3 render test 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#f2e9e1",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#7f58ff",
        "fontSize": 50,
        "fontWeight": "bold",
        "marginBottom": 40,
      }
    }
  >
    therAPPy
  </Text>
  <Text
    style={
      Object {
        "color": "black",
        "fontSize": 18,
        "marginBottom": 10,
        "marginLeft": 10,
        "marginRight": 10,
      }
    }
  >
    To start off you will take a survey, the survey operates like this:
  </Text>
  <Text
    style={
      Object {
        "color": "black",
        "fontSize": 18,
        "marginBottom": 10,
        "marginLeft": 10,
        "marginRight": 10,
      }
    }
  >
    Rate the level of agreement with each statement by selecting a value from 0 to 4
  </Text>
  <View
    style={
      Array [
        undefined,
        undefined,
        Object {
          "flexDirection": "row",
        },
      ]
    }
  >
    <View>
      <View
        style={
          Array [
            Object {
              "flexDirection": "row",
              "marginBottom": 5,
            },
            undefined,
            Object {
              "flexDirection": "column",
              "paddingLeft": 10,
            },
          ]
        }
      >
        <View>
          <View
            accessibilityLabel="radioButtonInput0"
            accessible={true}
            focusable={true}
            isTVSelectable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "borderColor": "#2196f3",
                "borderRadius": 15,
                "borderWidth": 3,
                "height": 30,
                "justifyContent": "center",
                "opacity": 1,
                "width": 30,
              }
            }
            testID="radioButtonInputradioButton|0"
          >
            <View
              style={
                Array [
                  Object {
                    "borderRadius": 10,
                  },
                  Object {
                    "backgroundColor": "#2196f3",
                    "height": 20,
                    "width": 20,
                  },
                  Object {
                    "borderRadius": 10,
                    "height": 20,
                    "width": 20,
                  },
                  Object {
                    "backgroundColor": "#2196f3",
                  },
                ]
              }
            />
          </View>
        </View>
        <View
          accessibilityLabel="radioButtonLabel0"
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              undefined,
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "flexDirection": "row",
              },
            ]
          }
          testID="radioButtonLabelradioButton|0"
        >
          <Text
            style={
              Array [
                Object {
                  "lineHeight": 20,
                  "paddingLeft": 10,
                },
                Object {
                  "paddingLeft": 0,
                },
                Object {
                  "color": "#000",
                },
                undefined,
              ]
            }
          >
            0
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        style={
          Array [
            Object {
              "flexDirection": "row",
              "marginBottom": 5,
            },
            undefined,
            Object {
              "flexDirection": "column",
              "paddingLeft": 10,
            },
          ]
        }
      >
        <View>
          <View
            accessibilityLabel="radioButtonInput1"
            accessible={true}
            focusable={true}
            isTVSelectable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "borderColor": "#2196f3",
                "borderRadius": 15,
                "borderWidth": 3,
                "height": 30,
                "justifyContent": "center",
                "opacity": 1,
                "width": 30,
              }
            }
            testID="radioButtonInputradioButton|1"
          >
            <View
              style={
                Array [
                  Object {
                    "borderRadius": 10,
                  },
                  false,
                  false,
                  false,
                ]
              }
            />
          </View>
        </View>
        <View
          accessibilityLabel="radioButtonLabel1"
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              undefined,
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "flexDirection": "row",
              },
            ]
          }
          testID="radioButtonLabelradioButton|1"
        >
          <Text
            style={
              Array [
                Object {
                  "lineHeight": 20,
                  "paddingLeft": 10,
                },
                Object {
                  "paddingLeft": 0,
                },
                Object {
                  "color": "#000",
                },
                undefined,
              ]
            }
          >
            1
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        style={
          Array [
            Object {
              "flexDirection": "row",
              "marginBottom": 5,
            },
            undefined,
            Object {
              "flexDirection": "column",
              "paddingLeft": 10,
            },
          ]
        }
      >
        <View>
          <View
            accessibilityLabel="radioButtonInput2"
            accessible={true}
            focusable={true}
            isTVSelectable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "borderColor": "#2196f3",
                "borderRadius": 15,
                "borderWidth": 3,
                "height": 30,
                "justifyContent": "center",
                "opacity": 1,
                "width": 30,
              }
            }
            testID="radioButtonInputradioButton|2"
          >
            <View
              style={
                Array [
                  Object {
                    "borderRadius": 10,
                  },
                  false,
                  false,
                  false,
                ]
              }
            />
          </View>
        </View>
        <View
          accessibilityLabel="radioButtonLabel2"
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              undefined,
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "flexDirection": "row",
              },
            ]
          }
          testID="radioButtonLabelradioButton|2"
        >
          <Text
            style={
              Array [
                Object {
                  "lineHeight": 20,
                  "paddingLeft": 10,
                },
                Object {
                  "paddingLeft": 0,
                },
                Object {
                  "color": "#000",
                },
                undefined,
              ]
            }
          >
            2
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        style={
          Array [
            Object {
              "flexDirection": "row",
              "marginBottom": 5,
            },
            undefined,
            Object {
              "flexDirection": "column",
              "paddingLeft": 10,
            },
          ]
        }
      >
        <View>
          <View
            accessibilityLabel="radioButtonInput3"
            accessible={true}
            focusable={true}
            isTVSelectable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "borderColor": "#2196f3",
                "borderRadius": 15,
                "borderWidth": 3,
                "height": 30,
                "justifyContent": "center",
                "opacity": 1,
                "width": 30,
              }
            }
            testID="radioButtonInputradioButton|3"
          >
            <View
              style={
                Array [
                  Object {
                    "borderRadius": 10,
                  },
                  false,
                  false,
                  false,
                ]
              }
            />
          </View>
        </View>
        <View
          accessibilityLabel="radioButtonLabel3"
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              undefined,
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "flexDirection": "row",
              },
            ]
          }
          testID="radioButtonLabelradioButton|3"
        >
          <Text
            style={
              Array [
                Object {
                  "lineHeight": 20,
                  "paddingLeft": 10,
                },
                Object {
                  "paddingLeft": 0,
                },
                Object {
                  "color": "#000",
                },
                undefined,
              ]
            }
          >
            3
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        style={
          Array [
            Object {
              "flexDirection": "row",
              "marginBottom": 5,
            },
            undefined,
            Object {
              "flexDirection": "column",
              "paddingLeft": 10,
            },
          ]
        }
      >
        <View>
          <View
            accessibilityLabel="radioButtonInput4"
            accessible={true}
            focusable={true}
            isTVSelectable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "borderColor": "#2196f3",
                "borderRadius": 15,
                "borderWidth": 3,
                "height": 30,
                "justifyContent": "center",
                "opacity": 1,
                "width": 30,
              }
            }
            testID="radioButtonInputradioButton|4"
          >
            <View
              style={
                Array [
                  Object {
                    "borderRadius": 10,
                  },
                  false,
                  false,
                  false,
                ]
              }
            />
          </View>
        </View>
        <View
          accessibilityLabel="radioButtonLabel4"
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              undefined,
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "flexDirection": "row",
              },
            ]
          }
          testID="radioButtonLabelradioButton|4"
        >
          <Text
            style={
              Array [
                Object {
                  "lineHeight": 20,
                  "paddingLeft": 10,
                },
                Object {
                  "paddingLeft": 0,
                },
                Object {
                  "color": "#000",
                },
                undefined,
              ]
            }
          >
            4
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "space-evenly",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "marginLeft": 90,
          "marginTop": 25,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#7f58ff",
            "borderRadius": 10,
            "height": 40,
            "justifyContent": "center",
            "opacity": 1,
            "width": "45%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "black",
              "fontSize": 18,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginRight": 10,
            }
          }
        >
          Back!
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "flex": 1,
          "marginTop": 25,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#7f58ff",
            "borderRadius": 10,
            "height": 40,
            "justifyContent": "center",
            "opacity": 1,
            "width": "45%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "black",
              "fontSize": 18,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginRight": 10,
            }
          }
        >
          Next!
        </Text>
      </View>
    </View>
  </View>
</View>
`;
